source/routing/raptor.cpp

```diff
@@ -322,12 +442,29 @@ RAPTOR::compute_all(const map_stop_point_duration& departures_,
         const auto reader_results = read_solutions(*this,
                                                    !clockwise,
                                                    departure_datetime,
-                                                   departures_,
+                                                   departures,
                                                    destinations,
                                                    rt_level,
                                                    accessibilite_params);
-        for (const auto& s: reader_results) { solutions.add(s); }
+        bool has_added = false;
+        for (const auto& s: reader_results) { has_added = (solutions.add(s) || has_added); }
```

Comment:
I don't think it's possible as I want to insert all `s`'s and know if one insertion was `true`
But the code is not straightforward and there might be a way to improve it